client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MlClientDocumentationIT.java

```diff
@@ -749,6 +750,10 @@ public void testUpdateDatafeed() throws Exception {
             DatafeedConfig updatedDatafeed = response.getResponse(); // <1>
             // end::update-datafeed-response
             assertThat(updatedDatafeed.getId(), equalTo(datafeedId));
+        } catch (WarningFailureException e) {
```

Comment:
We know this will always throw so the assertion on on line 752 will never be tested. 

Since we are deprecating setting the job Id why not remove it from the update i.e. delete line 734 
```
.setJobId("update-datafeed-job"); // <10>
```

And remove it from `update-datafeed.asciidoc`

Reply:
By `update-datafeed.asciidoc` I mean the REST high level client version